src/Courses/TopicSettings/RendererPreview.tsx

```diff
@@ -23,6 +23,7 @@ export const RendererPreview: React.FC<RendererPreviewProps> = ({defaultPath}) =
     useEffect(()=>{
         if (!_.isNil(defaultPath)) {
             setPreviewSettings(settings => ({...settings, path: defaultPath}));
+            forceUpdate();
```

Comment:
What does this do?

Reply:
This forces an update.

Reply:
Specifically, it generates a new dummy problem object to force the iframe to rerender. A hack to not have to genericize the ProblemIframe component.

Reply:
causes the preview pane to refresh with the new problem path.